lib/ui/painting.dart

```diff
@@ -1340,6 +1347,18 @@ enum PathFillType {
   evenOdd,
 }
 
+/// Path operation enums
+/// 
+/// For use with [Path.combine]
+/// Must be kept in sync with SkPathOp
+enum PathOperation {
+    difference,         //!< subtract the op path from the first path
```

Comment:
These should have documentation on the line before them, with a blank line between them, and indented by two not four. Also, the descriptions should be complete sentences, maybe with examples.

```
  /// Subtract the second path from the first path.
  ///
  /// For example, if the first path is a circle, and the second path is a smaller
  /// circle in the center of the first, then the result is a doughnut. 
  difference,

  /// Intersect the two paths.
  ///
  /// For example, if the paths are a long horizontal rectangle intersecting a long
  /// vertical rectangle with the same width, the result will be a square, where they
  /// would overlap.
  intersect,
```
...and so on.